homeassistant/components/shelly/binary_sensor.py

```diff
@@ -206,6 +206,13 @@ class RestBinarySensorDescription(RestEntityDescription, BinarySensorEntityDescr
         name="Smoke",
         device_class=BinarySensorDeviceClass.SMOKE,
     ),
+    "restart": RpcBinarySensorDescription(
```

Comment:
Do we need this sensor to be enabled for all installations? The restart required is a very rare case

Reply:
Good point, let me move it to disabled